src/main.c

```diff
@@ -20,11 +20,18 @@ int _write(int file, char *ptr, int len);
  * This output frequency is possible thanks to the Phase Locked Loop (PLL)
  * multiplier.
  *
+ * The default values for preescalers are:
+ * ADC prescaler(Max. 14MHz): Divider 8 / Set 9MHz
+ * APB1 prescaler(Max. 36MHz): Divider 2 / Set 36MHz
+ * APB2 prescaler(Max. 72MHz): Divider 1 / Set 72 MHz
+ * AHB prescaler(Max. 72MHz): Divider 1 / Set 72 MHz
+
```

Comment:
Not sure this should go here. Maye a "@see" note to point to the reference manual section? Or maybe better in Athena project?

Anyway if it is a list, I would format it like:

```c
/**
 * [...]
 * 
 * The default values for the prescalers are:
 *
 * - ADC prescaler (max. 14 MHz): divider 8 / set 9 MHz.
 * - ...
```

Note the space before parenthesis, new line before list, no caps if no dot/stop before the word and a space before units (I think the only exception is for degrees).

What does "divider X / set Y" means here? Maybe it is not clear. :sweat_smile:

Reply:
Included a @see section with the reference manual, and comment rewritten. Without issues I hope :)